File: tamgiac.py
def classify_triangle(a, b, c):
    if a + b > c and a + c > b and b + c > a:
        if a == b and b == c:
            return "Equilateral"
        elif a == b or b == c or c == a:
            return "Isosceles"
        else:
            return "Scalene"
    else:
        return "NotATriangle"

a = int(input("Nhập cạnh a: "))
b = int(input("Nhập cạnh b: "))
c = int(input("Nhập cạnh c: "))
print(classify_triangle(a, b, c))

File: tamgiac.java
import java.util.Scanner;

public class tamgiac {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Nhập cạnh a: ");
        int a = scanner.nextInt();
        System.out.println("Nhập cạnh b: ");
        int b = scanner.nextInt();
        System.out.println("Nhập cạnh c: ");
        int c = scanner.nextInt();
        System.out.println(classifyTriangle(a, b, c));
    }

    public static String classifyTriangle(int a, int b, int c) {
        if (a + b > c && a + c > b && b + c > a) {
            if (a == b && b == c) {
                return "Equilateral";
            } else if (a == b || b == c || c == a) {
                return "Isosceles";
            } else {
                return "Scalene";
            }
        } else {
            return "NotATriangle";
        }
    }
}
